src/components/Global/SearchAndFilter/SearchAndFilter.module.scss

```diff
@@ -0,0 +1,8 @@
+.container {
+  display: flex;
+  justify-content: space-between;
+}
+
+.input {
+  width: 26.75rem;
```

Comment:
Do we need to manually fix this width? It's usually bad for responsiveness to do this